client/object_replicate_test.go

```diff
@@ -26,11 +26,7 @@ func BenchmarkPrepareReplicationMessage(b *testing.B) {
 	id := oidtest.ID()
 
 	var signer nopSigner
-
-	b.ReportAllocs()
```

Comment:
I hate `b.ReportAllocs()`, but that's a different problem, not related to `b.Loop()`